kazoo/handlers/utils.py

```diff
@@ -241,7 +243,7 @@ def create_tcp_connection(
             # verify_mode to CERT_NONE.
             # TODO: Make hostname verification configurable as some users may
```

Comment:
Can we remove this TODO since your changes make it completed?

Reply:
Done :)